src/include/migraphx/dyn_output.hpp

```diff
@@ -21,44 +21,55 @@
  * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
  * THE SOFTWARE.
  */
-#ifndef MIGRAPHX_GUARD_RTGLIB_ELU_HPP
```

Comment:
not sure why this is showing up as changes to the elu.hpp file. If it will affect commit history, I would prefer the original elu.hpp file is removed, and the dyn_output.hpp file is added as a new file.

Reply:
I also have no idea. Does anyone know how to fix this? The original file should be deleted.

Reply:
Its a git thing, if a file looks similar enough git assumes it was renamed, unfortunately.